wmidump.c

```diff
@@ -200,26 +200,27 @@ static void *read_wdg(int fd, size_t *len)
 
 int main(void)
 {
+	void *data = NULL;
+	void *wdg = NULL;
 	size_t len;
-	void *data, *wdg;
 	int err = 0;
 
 	wdg = read_wdg(STDIN_FILENO, &len);
-	if (!wdg) {
-		err = -1;
```

Comment:
In general the commit messages in the series state WHAT is being done, but often omit WHY it's being done. In this case, why return 1 rather than -1. Without the WHY, it looks like bike shedding (although I suspect that is NOT the case, it is important to state your rationale in the commit message).